PYME/LMVis/Extras/snapshot.py

```diff
@@ -63,4 +63,4 @@ def save_snapshot(canvas):
 
 
 def Plug(vis_fr):
-    vis_fr.AddMenuItem('View', 'Save Snapshot', lambda e: save_snapshot(vis_fr.glCanvas))
+    vis_fr.AddMenuItem('File>Save', 'View Snapshot', lambda e: save_snapshot(vis_fr.glCanvas))

```

Comment:
I'm inclined to keep this on view - to me the File menu is for data / quantitative stuff and a snapshot is not data

Reply:
I get that, but I think if we put ourselves in the shoes of a user it's pretty weird to open a program with visualize in the name and not be able to save your visualization in the main 'file' menu. What if you opened Word and could only save to pdf from the view menu?

Reply:
I also think we get the snapshot-is-not-data point across by calling it a 'snapshot', or in this PR, 'view snapshot'.